JSDemos/Demos/Scheduler/SignalRService/description.md

```diff
@@ -1,3 +1,15 @@
-This demo shows how two users can edit data in&nbsp;the Scheduler simultaneously. A&nbsp;SignalR service is&nbsp;used to&nbsp;broadcast push notifications.
+This demo shows how you can use a <a href="https://dotnet.microsoft.com/apps/aspnet/signalr" target="blank">SignalR</a> service to synchronize appointments across different devices. Each Scheduler is bound to an individual store to emulate the situation when Schedulers are being edited on two different clients. Changes made in one Scheduler are repeated in the other and persist until the browser session has expired.
 
-Each Scheduler has an&nbsp;individual store to&nbsp;emulate the situation when **Schedulers** are being edited on&nbsp;two different clients. Changes made in&nbsp;one Scheduler are repeated in&nbsp;the other and persist until the browser session has expired.
\ No newline at end of file
+To implement this functionality, follow the steps below:
+
+1. Configure CustomStores. In this demo, we use the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data/blob/master/docs/client-side-with-jquery.md#api-reference" target="_blank">createStore</a> method (part of the <a href="https://github.com/DevExpress/DevExtreme.AspNet.Data" target="_blank">DevExtreme.AspNet.data</a> extension).
+
+1. Create store instances&mdash;one per Scheduler.
```

Comment:
In the last step, when I used the `createStore` method, didn't I already create a store instance? Judging by code, I guess what you mean here is "store a reference to the store in a variable". Some implementations differ depending on technology... But in any event it seems that these two steps are saying the same thing.